precompiles/conviction-voting/src/lib.rs

```diff
@@ -57,6 +57,14 @@ type ClassOf<Runtime> = <<Runtime as pallet_conviction_voting::Config>::Polls as
 pub(crate) const SELECTOR_LOG_VOTED: [u8; 32] =
 	keccak256!("Voted(uint32,address,bool,uint256,uint8)");
 
+/// Solidity selector of the Vote Split log, which is the Keccak of the Log signature.
+pub(crate) const SELECTOR_LOG_VOTE_SPLIT: [u8; 32] =
+	keccak256!("VoteSplit(uint32,address,uint256,uint256)");
+
+/// Solidity selector of the Vote Split Abstained log, which is the Keccak of the Log signature.
+pub(crate) const SELECTOR_LOG_VOTE_SPLIT_ABSTAINED: [u8; 32] =
+	keccak256!("VoteSplitAbstained(uint32,address,uint256,uint256,uint256)");
+
```

Comment:
Shouldn't these new events also be defined in precompiles/conviction-voting/ConvictionVoting.sol ? I only see the new functions there.

Reply:
thanks, added in https://github.com/PureStake/moonbeam/pull/2169/commits/00888191dbfee512828f56477f96f8dc960c3381